FirebaseStorageSwift/Tests/Integration/StorageAsyncAwait.swift

```diff
@@ -17,8 +17,8 @@ import FirebaseCore
 import FirebaseStorage
 import XCTest
 
-#if swift(>=5.9)
-  @available(iOS 15, *)
+#if swift(>=5.5)
```

Comment:
Does this need `&& canImport(_Concurrency)` too?

Reply:
Thanks! Technically, It might only be needed for Xcode 13.0 which we won't support in v9 - but I added it for consistency.